pwndbg/chain.py

```diff
@@ -138,7 +139,7 @@ def format(
     arrow_right = c.arrow(f" {config_arrow_right} ")
 
     # Colorize the chain
-    rest = [M.get_address_and_symbol(link) for link in chain]
+    rest = [M.get_address_and_symbol(addr) if addr >= 0 else "" for addr in chain]
```

Comment:
Lmao if `limit == 0` we will still colorize the whole chain even though we return `rest[0]`